### DISCUSSION TITLE
OSX m2 conda pip workaround

### DISCUSSION THREAD

**Discussion by KoSv** [cedricp/ddt4all#873] [Q&A]
Hey 

I had problems with getting it running/compiled on OSX M2.
This is what helped me out:

```
conda create -n <name>
conda activate <name>
conda config --env --set subdir osx-64
conda install python=3.8
```

**Comment by Furtif** [comment]
for MacOs try https://drive.google.com/file/d/1y46RNGut0HuXhUXsYMMo69qQ0-t7Nv9L/view?usp=drive_link

**Comment by Furtif** [comment]
or uses brew install python3 etc...

**Comment by Furtif** [comment]
if fine mark this solved
